README.md

```diff
@@ -1 +1,34 @@
 # SOM_Weather_Forecast_API
+
+###Description 
+This weather forecast application was created to conduct unit tests on the weather Api.
+
+
+##Languages used 
+
+#The language used for this project was Ruby.
+#The gems required for this project were http,json, and rspec.
+
+### How to download
+1. git clone the the repo unto your terminal and then save it or download it as a zip file on your computer.
```

Comment:
This does make sense `unto`. And then save it doesnt make sense. What do you mean?